packages/adapter-utils/src/utils.ts

```diff
@@ -229,8 +238,12 @@ const recurseIntoValue = ({
       // but we do not want to omit those, we only want to omit empty
       // objects, arrays and strings. we don't need to check for objects here
       // because we cannot get here with an object
-      const valueIsEmpty = (Array.isArray(newVal) || _.isString(newVal)) && _.isEmpty(newVal)
-      return valueIsEmpty && !allowEmpty ? undefined : newVal
+      const isEmptyString = _.isString(newVal) && _.isEmpty(newVal)
+      if (isEmptyString) {
+        log.warn('found empty string found in path %s, string will be omitted: %s', keyPathID, !allowEmptyArrays)
```

Comment:
`keyPathID` is an elem ID
this should be `keyPathID.getFullName()`
also
```suggestion
log.warn('found empty string in path %s, string will be omitted: %s', keyPathID.getFullName(), !allowEmptyArrays)
```

Reply:
thanks 🙏 
opened - https://github.com/salto-io/salto/pull/6075